packages/flutter/test/rendering/editable_test.dart

```diff
@@ -1659,9 +1659,9 @@ void main() {
       composingRect = editable.getRectForComposingRange(const TextRange(start: 6, end: 7))!;
       expect(composingRect, const Rect.fromLTRB(0.0, 14.0, 14.0, 28.0));
       composingRect = editable.getRectForComposingRange(const TextRange(start: 7, end: 8))!; // H
-      expect(composingRect, const Rect.fromLTRB(14.0, 18.0, 24.0, 28.0));
```

Comment:
This test change makes sense since now that the default is `BoxHeightStyle.max`, these rects have a height of the largest span on the line which are the `WidgetSpan`s in this test with a height of `14.0`.